addons/point_of_sale/static/src/app/screens/payment_screen/payment_screen.js

```diff
@@ -245,8 +245,6 @@ export class PaymentScreen extends Component {
             );
             return;
         }
-
-        pLine.setAmount(pLine.getAmount() - (tip || 0) + parseFloat(newTip));
```

Comment:
IMO It kinda make sense to add it to the payment line so that the cashier doesn't need to add another payment line.
Maybe we could compute the missing amount to match the tip amount instead of adding the whole tip amount ?